components/31-molecules/radios/README.md

```diff
@@ -0,0 +1,26 @@
+# Radios
+
+## When to use this component
+
+Use the radios component when users should only be able to select one option and no more.
+
+## When not to use this component
```

Comment:
You could reference the select component here for the use of a long list of options (and only 1 can be selected)

Reply:
Thanks for the suggestion! I added a reference to the select component.